webkit_server.py

```diff
@@ -513,12 +520,47 @@ def issue_command(self, cmd, *args):
     self._writeline(cmd)
     self._writeline(str(len(args)))
     for arg in args:
-      arg = str(arg)
-      self._writeline(str(len(arg)))
-      self._sock.sendall(arg.encode("utf-8"))
+        self.send_arg(arg)
 
     return self._read_response()
 
+  def send_arg(self, arg):
+    """ Send each arg for args in issue_command """
+    arg = self.prepare_for_socket_sendall(arg)
+    self._writeline(str(len(arg)))
+    self._sock.sendall(arg)
+
+  def prepare_for_socket_sendall(self, arg):
+    """ Deal with the unicode and bytes problem in Python 2 and Python 3 for socket.sendall """
+
+    if PY2:
+      if type(arg) != unicode:
```

Comment:
I am not sure if this raise an NameError in Python 3 since there's no `unicode` type in Python 3.
But I've tested it in Python 3 and it worked.
Would be appreciate if anyone has some advices on this.